modules/gmc-agent/src/main/java/org/gridgain/agent/StompDestinationsUtils.java

```diff
@@ -120,7 +98,7 @@ public static String buildActionRequestTopic(UUID clusterId) {
     /**
      * @return Add cluster destination.
```

Comment:
Missing `@param clusterId`
Change `Add cluster destination.` to `Send cluster info destination.`

Reply:
Changed